### PR TITLE
GUI - Integration REWORK

### PR DESCRIPTION
**Pull request by yef3**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -195,3 +195,259 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+DataBase/libhv/.clang-format
```
Comment (reC4P7CH4) [inline .gitignore:198]: No need to include `DataBase/libhv`, already ignored.

**Review by reC4P7CH4 — COMMENTED** [review]
Looks good! Let other people Approve.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ######   Project Specific   ######
-DataBase/libhv**
 ecdb
```
Comment (reC4P7CH4) [inline .gitignore:2]: Blocking Comment to Prevent Auto-Merge. Resolve When Ready to Merge.

**Review by reC4P7CH4 — APPROVED** [review]

**Review by Viyerelu23333 — APPROVED** [review]
好耶

**Pull request merged**